.golangci.yaml

```diff
@@ -0,0 +1,33 @@
+# options for analysis running
+run:
+  timeout: 2m
+  skip-files:
+  - "zz_generated.*"
+linters:
+  enable:
+  - gocritic
+  - godot
+  - gofumpt
+  - goimports
+  - misspell
+  - nolintlint
+  - predeclared
+  - revive
+  - unused
+  - stylecheck
+  - unconvert
+  - wastedassign
+  - whitespace
+  # - wsl
+linters-settings:
+  goimports:
+    local-prefixes: github.tools.sap/kubernetes/gardener-installer
```

Comment:
This does not seem like a correct prefix.

Reply:
Oops, thanks for catching! Fixed.